### PR TITLE
Changes and notes from Julie.

### PR DESCRIPTION
**Pull request by juliepixels**
I've added comments labeled NOTES on the areas I've changes. Let me know if I can answer any questions about the changes I proposed. These files are not the same ones that we updated on your computer so you may want to add your files first.

### CODE REVIEW

File: Final/About.html [hunk 1]
```diff
@@ -1,59 +1,50 @@
...
+<html lang="en">
 <head>
   <title>About</title>
   <link rel="stylesheet" href="styles.css">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 </head>
 <body>
-    <div>
-        <img src="./img/sweets.jpg" width="200px" style="float:right" alt="picture of my kids">
-    </div>
```
Comment (juliepixels) [inline Final/About.html:11]: I moved this image below the navigation and combined with in the `header` div.

File: Final/About.html [hunk 2]
```diff
@@ -1,59 +1,50 @@
...
-  <a href="./mystory.html" id="mystory">My Story</a>
-  <a href="./About.html" id="about">About</a>
-  <a href="./Resume.html" id="resume">Resume</a>
-  <a href="./teams.html" id="links">Links</a>
-  <a href="./Contact.html" id="contact">Contact</a>
-</div>
-</div><div style="margin-left:80px;">
- <section class="text-container">
-  <p> As a child, growing up in Ethiopia, we didn't have a computer at all and had no idea what exactly computer was. In 2006, with my three
-  siblings and our father, we moved to Canada to live with our stepmom. I went to high school starting from grade 9. One of my classes was a computer class which was a mandatory.
```
Comment (juliepixels) [inline Final/About.html:29]: Closed the `p` tag here.

File: Final/About.html [hunk 3]
```diff
@@ -1,59 +1,50 @@
...
+
+    <p>Since I became stay-home mother, my flexibility for learning and working has became too tight. My family member told me about Career Hub when I was looking for a part-time job. Fortunately, I found out about this amazing computer coding program at the Career Hub given by very amazing, helpful, and caring instructor and TAs. I was very excited and joined the team without hesitation. I have been having some obstacles attending the class but, I have promised not to give up and try my best as much as I can to complete and be proud of myself. Fortunately, everyone in this program has been very helpful which keeps my hopes up all the time. I have came to this so far, learned what html, css, and java scrip is and what they also do. I am still learning and knowing what it is like to be a web developer.</p>
   </section>
+
+  <div>
     <div>
-        <span><a href="https://github.com/UmmuAyyub" target="_blank" style="color:red">GitHub</a></span><br>
-        <span><a href="https://www.linkedin.com/in/nezira-usmael-6358b3182/" target="_blank" style="color:red">linkedin</a></span>
-        </div>
-    <section></section><br><br>
```
Comment (juliepixels) [inline Final/About.html:51]: Using `div` instead of `span` means you don't need `br` tags.

File: Final/Contact.html [hunk 4]
```diff
@@ -1,75 +1,89 @@
...
-  <a href="./Resume.html" id="resume">Resume</a>
-  <a href="./teams.html" id="links">Links</a>
-  <a href="./Contact.html" id="contact">Contact</a>
-</div>
+  <title>Contact Nezira Usmael</title><!-- NOTES: I added a title here. This is important since it helps people know what this page is about. -->
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="styles.css">
+</head><!-- NOTES: In each html document there can be only one <head> and one <body> tag. Just like for people :) -->
+
```
Comment (juliepixels) [inline Final/Contact.html:9]: Comments labeled NOTES let you know what I was thinking.

File: Final/Resume.html [hunk 5]
```diff
@@ -1,41 +1,38 @@
...
+  <link rel="stylesheet" href="styles.css">
 </head>
-<meta name="viewport" content="width=device-width, initial-scale=1">
-<link rel="stylesheet" href="styles.css">
-</head>
-<body>
-<div class="header">
-  <h1>Here is my CV!</h1>
-</div>
-
```
Comment (juliepixels) [inline Final/Resume.html:12]: Moved the `header` below the navigation.

File: Final/index.html [hunk 6]
```diff
@@ -1,16 +1,13 @@
...
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <style>
-   
-  </style>
 </head>
-
 <body>
+ <!-- NOTES: I added space between each main <div> just to make it easier to read. I also add indentions to the code to make it easier to read the html. -->
+
   <div id="mySidenav" class="sidenav">
```
Comment (juliepixels) [inline Final/index.html:11]: Lot's of changes. Comments labeled NOTES let you know what I was thinking.

File: Final/mystory.html [hunk 7]
```diff
@@ -1,52 +1,54 @@
 <html>
-<head>
-<title>My Story</title>
-</head>
-<meta charset="utf-8">
-<meta name="viewport" content="width=device-width, initial-scale=1">
-<link rel="stylesheet" href="styles.css">
-</head>
-<body> 
-</body>
```
Comment (juliepixels) [inline Final/mystory.html:10]: There can only be one `head` and `body` per page.

File: Final/styles.css [hunk 8]
```diff
@@ -4,7 +4,8 @@ p {
   color: white;
 }
```
Comment (juliepixels) [inline Final/styles.css:6]: Comments labeled NOTES let you know what I was thinking.

File: Final/About.html (hunk 2 above)
Comment (UmmuAyyub) [reply Final/About.html:29]: Thank you Julie.

File: Final/About.html (hunk 3 above)
Comment (UmmuAyyub) [reply Final/About.html:51]: Thank you Julie.

**Pull request closed without merging**